config/config.go

```diff
@@ -539,6 +535,17 @@ func (r PortRange) Match(port int) bool {
 	return r.From <= port && r.To >= port
 }
 
+type PortsWhiteList map[string]PortWhiteList
+
+func (pwl *PortsWhiteList) Decode(value string) error {
+	err := json.Unmarshal([]byte(value), pwl)
+	if err != nil {
+		log.Error("Error unmarshalling TYK_GW_PORTWHITELIST")
```

Comment:
@buraksekili @mativm02 I think it would be a good idea to return an error here and print the unmarshal err.